src/components/shared/Entity/EndpointConfig/Header.tsx

```diff
@@ -29,12 +28,12 @@ function EndpointConfigHeader({ docsPath }: Props) {
                 <FormattedMessage id="entityCreate.endpointConfig.heading" />
             </Typography>
 
-            {docsPath && docsPath.length > 0 ? (
+            {belowMd && docsUrl && docsUrl.length > 0 ? (
```

Comment:
We now can fetch the URL from the store now and then we just show for belowMd because then we just skip showing the docs inline.